package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "esri-loader",
-  "version": "2.0.0",
+  "version": "2.1.0",
```

Comment:
This will need to be reverted - the it is handled by our release script. Generally it's a good idea to leave version bumps out of PRs.